server/chat-plugins/thing-of-the-day.ts

```diff
@@ -50,7 +71,7 @@ class OtdHandler {
 	lastPrenom: number;
 	winners: AnyObject[];
 	constructor(
-		id: string, name: string, room: ChatRoom, filename: string, keys: string[], keyLabels: string[], week = false
+		id: string, name: string, room: ChatRoom, filename: string,	keys: string[], keyLabels: string[], week = false,
```

Comment:
![image](https://user-images.githubusercontent.com/551184/96418905-42edf100-11a8-11eb-8ee1-62409c32f438.png)

This part is kinda ominous...

Reply:
Oh, it looks like you changed a space into a tab. I'm surprised that passes CI.

```suggestion
		id: string, name: string, room: ChatRoom, filename: string, keys: string[], keyLabels: string[], week = false,
```

Reply:
The trailing comma is also wrong, though; why isn't the validator picking up on that?